### DISCUSSION TITLE
Postgres warning "pg_stat_tmp/global.stat"

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#170] [Q&A]
When starting mediacms_db_1 postgres, the following warning appears.

> WARNING:  could not open statistics file "pg_stat_tmp/global.stat": Operation not permitted

Running docker on a mac, Big Sur 11.2.1.

What is this warning?

**Comment by OccultWarlock** [comment]
Were you ever able to figure out what this error is?
